### PR TITLE
Mw 1227/change db driver to pgx

### PR DESCRIPTION
**Pull request by Moon1it**
https://github.com/tritonJS826/masters-way/issues/1227

### CODE REVIEW

File: mw-server/db_pgx/query/job_tags.query.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+-- name: GetJobTagByUuid :one
+SELECT * FROM job_tags
+WHERE job_tags.uuid = $1;
```
Comment (tritonJS826) [inline mw-server/db_pgx/query/job_tags.query.sql:3]: Let's use syntax 
```suggestion
WHERE job_tags.uuid = @job_tags;
```

File: mw-server/db_pgx/query/job_dones_job_tags.query.sql [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+--         WHERE job_dones.uuid = job_dones_job_tags.job_done_uuid
+--     ),
+--     '{}'
+-- )::VARCHAR[] AS tag_uuids
+-- FROM job_dones WHERE job_dones.day_report_uuid = ANY($1::UUID[]);
+
+-- name: DeleteJobDonesJobTagByJobDoneId :exec
+-- DELETE FROM job_dones_job_tags
+-- WHERE job_done_uuid = $1
+-- AND job_tag_uuid = $2;
```
Comment (tritonJS826) [inline mw-server/db_pgx/query/job_dones_job_tags.query.sql:25]: Remove unused code

File: mw-server/db_pgx/query/job_tags.query.sql (hunk 1 above)
Comment (Moon1it) [reply mw-server/db_pgx/query/job_tags.query.sql]: Done

File: mw-server/db/query/job_dones.query.sql [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
+--     (SELECT name FROM users WHERE uuid = $5) AS owner_name,
+--     -- get tag uuids
+--     COALESCE(
+--         ARRAY(
+--             SELECT job_dones_job_tags.job_tag_uuid
+--             FROM job_dones_job_tags
+--             WHERE job_dones.uuid = job_dones_job_tags.job_done_uuid
+--         ),
+--     '{}'
+--     )::VARCHAR[] AS tag_uuids;
```
Comment (tritonJS826) [inline mw-server/db/query/job_dones.query.sql:21]:
```suggestion
```

**Review by tritonJS826 — APPROVED** [review]

**Pull request merged**